### PR TITLE
Speck app improvements

### LINKED ISSUE #247: Speck application is missing some features
Most other apps have sample data available for download and accept uploads.

### PR DESCRIPTION
**Pull request by shammamah**
Closes #247 

## About 
- [ ] This is a new component 
- [ ] I am adding a feature to an existing component, or improving an existing feature
- [x] I am closing an issue 

## Description of changes 
* Some restyling according to the general app stylesheet from #316. 
* Add upload functionality and sample data for download.

## Before merging
- [x] I have gone through the [code review checklist](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/review_checklist.md)
- [x] I have completed all of the [steps to take before merging](https://github.com/plotly/dash-bio/blob/master/.github/before_merging.md)
- [x] I know how to [deploy to DDS](https://github.com/plotly/dash-bio/blob/master/.github/deploying_to_dds.md)

### CODE REVIEW

File: assets/speck-style.css [hunk 1]
```diff
@@ -1,55 +1,13 @@
 #speck-body {
-    font-family: 'Open Sans', sans-serif;
-    padding: 5px;
-    background-color: #fff;
-    color: #232323;
-    font-weight:100;
-    height: calc(100vh - 65px);
-    overflow: auto;
+    background-color: #f;
```
Comment (mkcor) [inline assets/speck-style.css]:
```suggestion
    background-color: #FFFFFF;
```

File: assets/speck-style.css [hunk 2]
```diff
@@ -61,54 +19,49 @@
...
-    top: -10px;
-}
-#speck-preset-atom-style-dropdown, #speck-preset-rendering-dropdown {
+#speck-preset-atom-style-dropdown, #speck-preset-rendering-dropdown, #speck-molecule-dropdown {
+    padding-top: 5px;
     display: inline-block;
-    width: calc(100% - 150px);
-    vertical-align: middle;
+    float: right;
+    width: 220px; ;
```
Comment (mkcor) [inline assets/speck-style.css]:
```suggestion
    width: 220px;
```

File: tests/dashbio_demos/app_speck.py [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import os
+import base64
```
Comment (mkcor) [inline tests/dashbio_demos/app_speck.py]: You might as well skip a line after this import ;)

Comment (shammamah) [reply tests/dashbio_demos/app_speck.py]: Done in 99551cd!

Comment (mkcor) [reply tests/dashbio_demos/app_speck.py]: 'space' --> 'blank line' but ok :)

**Pull request merged**